test/general-tests/constructor-test/check.ts

```diff
@@ -1,4 +1,33 @@
-import { hardhatStarknetCompileDeprecated, hardhatStarknetTest } from "../../utils/cli-functions";
+import path from "path";
+import {
+    hardhatStarknetCompile,
+    hardhatStarknetCompileDeprecated,
+    hardhatStarknetTest
+} from "../../utils/cli-functions";
+import { copyFileSync } from "fs";
+import { assertContains, rmrfSync } from "../../utils/utils";
+
+const prefix = path.join(__dirname);
+const contract1 = "duplicate_constructor.cairo";
```

Comment:
Why not use descriptive variable naming? It could be `const duplicateConstructorContractName = "...`

Reply:
https://github.com/0xSpaceShard/starknet-hardhat-plugin/pull/377/commits/9b6048abf937d5c0936f1f5a4bf065acd4449935